django-backend/fecfiler/reports/migrations/0002_initial.py

```diff
@@ -1,24 +1,20 @@
-# Generated by Django 3.2.12 on 2022-09-15 17:09
```

Comment:
Can this migration file be squashed into the 0001_initial.py file?

Reply:
@mjtravers this particular one can't be squashed since reports has a circular dependency with web_service and the models might need reworked to avoid this.  I was however able to squash the memo_text 0002 migration that got generated for some reason.

Reply:
Sounds good. Thanks